src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -107,7 +181,7 @@ class UniversalNavbar extends React.Component {
             top: 0,
             left: 0,
             width: '100%',
-            height: 100,
+            height: 64,
```

Comment:
Fixes bug where fixed navbar is too tall and blocks on screen elements below the navbar. cc @iamvictorli this will fix not being able to click the package select back button on mobile for your CMS PR.